addons/mail/static/src/components/link_preview_card/link_preview_card.scss

```diff
@@ -0,0 +1,19 @@
+// ------------------------------------------------------------------
+// Layout
+// ------------------------------------------------------------------
+
+.o_LinkPreviewCard {
+    max-width: 33vw;
+}
+
+.o_LinkPreviewCard_image img {
+    max-height: 160px;
+}
+
+// ------------------------------------------------------------------
+// Style
+// ------------------------------------------------------------------
+
+.o_LinkPreviewCard:hover .o_LinkPreviewAside_itemUnlinkIcon {
+    visibility: visible;
+}
```

Comment:
Ok I see (related to above comment), but prefer modelling with state in JS rather than css specificity all around components.